qt/aqt/tts.py

```diff
@@ -516,3 +518,63 @@ def _rate_for_speed(self, speed: float) -> int:
             "eg. 1.5 -> 15, 0.5 -> -5"
             speed = (speed * 10) - 10
             return int(max(-10, min(10, speed)))
+
+    @dataclass
+    class WindowsRTVoice(TTSVoice):
+        id: Any
+
+    class WindowsRTTTSFilePlayer(TTSProcessPlayer):
+        voice_list = None
+        tmppath = os.path.join(tmpdir(), "tts.wav")
+
+        def import_voices(self) -> None:
+            import winrt.windows.media.speechsynthesis as speechsynthesis
+            self.voice_list = speechsynthesis.SpeechSynthesizer.get_all_voices()
+        def get_available_voices(self) -> List[TTSVoice]:
```

Comment:
Spaces between method names would be good - you can use 'bazel run qt:format' to automatically format the code.